src/python/sense/client/requestclient.py

```diff
@@ -0,0 +1,48 @@
+import json
+import requests
+from sense.common import evalInput
+from sense.client.mainclient import MainClient
+
+class RequestClient(MainClient):
+   def __init__(self):
+      super(RequestClient, self).__init__()
+
+   def get_service(self, tags):

```

Comment:
get/put/post/delete - lets name it `_get`/`_put`/`_post`/`_delete` and also have a new function request_wrapper(call_type, tags, **kwargs), where call_type is = GET/PUT/POST/DELETE, tags = URL; kwargs any other params, like data for requests.post